runatlantis.io/docs/server-configuration.md

```diff
@@ -927,6 +927,13 @@ and set `--autoplan-modules` to `false`.
   environment where releases.hashicorp.com is not available. Directory structure of the custom
   endpoint should match that of releases.hashicorp.com.
 
+### `--tf-disable-downloads`
+  ```bash
+  atlantis server --tf-disable-downloads
+  ```
+  Prevent Atlantis from trying to list and download additional versions of Terraform.
+  Useful in an airgapped environment where a download mirror is not available.
```

Comment:
Please confirm if you've tested this flag out and it works as expected when the flag is set and when the the env variable is used

Reply:
I had only tested this via a config file initially.

With this round of updates, I've tested this using the config file and environment variables. It seems to be correctly detecting the version in our projects when downloads are disabled (since we use exact version constraints). When downloads are enabled, I can validate that it attempts to list the versions, but because of our firewalling, it then dies because of the discussed `terraform-switcher` behaviour. Still, that shows that both configurations seem to be behaving as expected.

Reply:
I also verified that the `--tf-download-url` option now switches the URL that `terraform-switcher` uses for listing Terraform versions, so this should address #2799